### DISCUSSION TITLE
change kubelet root dir

### DISCUSSION THREAD

**Discussion by grk-pancham** [rancher/rke2#2436] [Q&A]
if we change the kubelet root directory (--root-dir) to some other directory other than the default /var/lib/kubelet,will kubelet monitor the new directory or the /var/lib/kubelet to apply node level eviction rules like nodefs etc ?

**Comment by brandond** [comment]
The k3s kubelet behavior in this regard is not changed from upstream. Whatever the normal Kubelet does, the K3s kubelet should do as well.
